core/src/main/java/com/amplifyframework/api/GraphQlBehavior.java

```diff
@@ -23,14 +23,13 @@
 import com.amplifyframework.api.graphql.GraphQLResponse;
 import com.amplifyframework.api.graphql.MutationType;
 import com.amplifyframework.api.graphql.SubscriptionType;
-import com.amplifyframework.core.ResultListener;
-import com.amplifyframework.core.StreamListener;
+import com.amplifyframework.core.Action;
+import com.amplifyframework.core.Consumer;
 import com.amplifyframework.core.model.Model;
 import com.amplifyframework.core.model.query.predicate.QueryPredicate;
-import com.amplifyframework.hub.HubCategory;
 
 /**
- * GraphQL behaviors which include operations like query/subscribe/mutate operations.
+ * GraphQL behaviors include operations like query, subscribe, mutate, etc.
```

Comment:
Is there any other GraphQL behavior besides those three specified?

Reply:
Will update.